docs/asciidoc/static/advanced-pipeline.asciidoc

```diff
@@ -116,6 +116,14 @@ filter {
 }
 --------------------------------------------------------------------------------
 
+To set the stdout output to json format:
```

Comment:
Can we be consistent with the following paragraph and say "JSON" instead of "json"?